src/components/VoteCard.vue

```diff
@@ -178,11 +178,13 @@ export default {
     loading: false,
     progressMessage: '',
     description: '',
+    stakeAmount: 0,
   }),
   computed: {
     ...mapState(['tokenInfo']),
   },
   async mounted() {
+    this.stakeAmount = +this.vote.stakeAmount;
```

Comment:
why the `+` here?

Reply:
Probably I had some precautions with shifting decimals which I can not remember.🤷🏻‍♂️
I just tested it and it's working just fine as a string, so indeed the ``+`` is unnecessary